src/EditorFeatures/Core/Implementation/IntelliSense/QuickInfo/IntellisenseQuickInfoBuilder.cs

```diff
@@ -47,18 +55,90 @@ internal static IntellisenseQuickInfoItem BuildItem(ITrackingSpan trackingSpan,
                 quickInfoItem.Sections.Where(s => s.Kind != QuickInfoSectionKinds.Description)
                                       .Select(BuildClassifiedTextElement));
 
+            // build text for RelatedSpan
+            if (quickInfoItem.RelatedSpans.Any())
+            {
+                elements.Add(await BuildTextElementForRelatedSpans(quickInfoItem, snapshot, document, cancellationToken).ConfigureAwait(false));
+            }
+
             var content = new ContainerElement(
                                 ContainerElementStyle.Stacked,
                                 elements);
 
             return new IntellisenseQuickInfoItem(trackingSpan, content);
         }
 
+        private static async Task<ClassifiedTextElement> BuildTextElementForRelatedSpans(CodeAnalysisQuickInfoItem quickInfoItem, ITextSnapshot snapshot, Document document, CancellationToken cancellationToken)
+        {
+            var classificationService = document.Project.LanguageServices.GetService<IClassificationService>();
```

Comment:
can this refactored a bit as generic helper method? by not tied to quickinfo?

Reply:
you could refactor and share code with this a bit
http://source.roslyn.io/#Microsoft.CodeAnalysis.EditorFeatures/FindUsages/ClassifiedSpansAndHighlightSpanFactory.cs,127

Reply:
document.GetLanguageService